packages/ffe-datepicker-react/src/calendar/Calendar.js

```diff
@@ -162,6 +164,18 @@ export default class Calendar extends Component {
         this.forceUpdate();
     }
 
+    selectYear(evt) {
+        const selectedYear = parseInt(evt.currentTarget.value);
```

Comment:
Det er god praxis og bruke radix
`const selectedYear = parseInt(evt.currentTarget.value, 10);`

Reply:
To make sure decimal numeral system is used. Got it.